package/libyui-qt.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Nov 29 14:42:52 UTC 2019 - Rodion Iafarov <riafarov@suse.com>
+
+- Add support to operate on many widgets with rest-api
+- 2.52.0
```

Comment:
Please also mention the so_version 11 bump.

Reply:
Done, thanks.